src/utils/fetch-twitch-videos.js

```diff
@@ -1,9 +1,13 @@
 import { get } from './request';
+import { TWITCH_HEADERS, TWITCH_VIDEO_URL } from '../constants';
 
 // Fetches videos from twitch api
-const fetchTwitchVideos = async (headers, video_URL, videoLimit = 1) => {
+const fetchTwitchVideos = async (
```

Comment:
Sorry if I was unclear on this, but I was thinking of having something like 

```
const fetchTwitchVideos = async (
videoLimit,
headers = TWITCH_HEADERS,
videoURL = TWITCH_VIDEO_URL
) => ...
```
and then doing the logic for `TWITCH_VIDEO_URL + '&first=' + videoLimit` in this function if `videoLimit` is not `null`, since this function should be responsible for handling that piece of the URL.

Does this make sense? That way we can clean up the use in `use-twitch-api` to be just `const videoRespData = await fetchTwitchVideos(videoLimit)`, and everywhere else just use `fetchTwitchVideos()` if we want them all.

Reply:
Aah I see! Sorry, I misunderstood earlier